bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -362,7 +362,7 @@ spec:
 
           hybrid_overlay_flags=
           if [[ -n "{{.OVNHybridOverlayEnable }}" ]]; then
-            hybrid_overlay_flags="--enable-hybrid-overlay"
+            hybrid_overlay_flags="--enable-hybrid-overlay --no-hostsubnet-nodes=\"kubernetes.io/os=windows\""
```

Comment:
@pecameron not your fault I know, but it turns out we don't need the escaped quotes here:

+ exec /usr/bin/ovnkube --init-master ip-10-0-144-152.us-east-2.compute.internal --config-file=/run/ovnkube-config/ovnkube.conf --ovn-empty-lb-events --loglevel 4 --enable-hybrid-overlay '--no-hostsubnet-nodes="kubernetes.io/os=windows"' --hybrid-overlay-cluster-subnets=10.132.0.0/14 --metrics-bind-address 0.0.0.0:9102 --sb-address ssl://10.0.130.21:9642,ssl://10.0.144.152:9642,ssl://10.0.171.9:9642 --sb-client-privkey /ovn-cert/tls.key --sb-client-cert /ovn-cert/tls.crt --sb-client-cacert /ovn-ca/ca-bundle.crt
time="2020-03-05T21:29:01Z" level=info msg="Parsed config file /run/ovnkube-config/ovnkube.conf"
time="2020-03-05T21:29:01Z" level=info msg="Parsed config: {Default:{MTU:8901 ConntrackZone:64000 EncapType:geneve EncapIP: EncapPort:6081 InactivityProbe:100000 OpenFlowProbe:180 RawClusterSubnets:10.128.0.0/14/23 ClusterSubnets:[]} Logging:{File:/dev/stdout Level:4} CNI:{ConfDir:/etc/cni/net.d Plugin:ovn-k8s-cni-overlay WinHNSNetworkID:} Kubernetes:{Kubeconfig: CACert: APIServer:https://api-int.aravindh-winc.devcluster.openshift.com:6443 Token: ServiceCIDR:172.30.0.0/16 OVNConfigNamespace:openshift-ovn-kubernetes MetricsBindAddress: MetricsEnablePprof:false OVNEmptyLbEvents:false PodIP: RawNoHostSubnetNodes: NoHostSubnetNodes:nil} OvnNorth:{Address: PrivKey: Cert: CACert: Scheme: northbound:false externalID: exec:<nil>} OvnSouth:{Address: PrivKey: Cert: CACert: Scheme: northbound:false externalID: exec:<nil>} Gateway:{Mode:local Interface: NextHop: VLANID:0 NodeportEnable:true} MasterHA:{ElectionLeaseDuration:60 ElectionRenewDeadline:30 ElectionRetryPeriod:20 ManageDBServers:false NbPort:6641 SbPort:6642}}"
time="2020-03-05T21:29:01Z" level=fatal msg="labelSelector \"\"kubernetes.io/os=windows\"\" is invalid: couldn't parse the selector string \"\"kubernetes.io/os=windows\"\": unable to parse requirement: invalid label key \"\\\"kubernetes.io/os\": prefix part a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')"